crdppf/lib/geometry_functions.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: UTF-8 -*-
+
+from crdppf.models import DBSession
+from crdppf.models import Property, PaperFormats
+
+from sqlalchemy.sql import func
+
+def get_feature_center(idemai):
+    """ Extract a feature centroid regarding its idemai attribute
+    """
+
+    geom = DBSession.query(Property.geom).filter(Property.idemai==idemai). \
```

Comment:
hmm, shouldn't we accept all property geom's in return and rather raise an error if there is more than one result (which shouldn't be the case)?